### PR TITLE
chore: use pnpm v8 for deps

### PR DESCRIPTION
**Pull request by raisedadead**
This PR bumps `pnpm` to v8

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Install peers for convenience
-auto-install-peers=true
```
Comment (scissorsneedfoodtoo) [inline .npmrc:2]: For future reviewers, it seems like the default for `auto-install-peers` is `true`, so it should be fine to remove here: https://pnpm.io/npmrc#auto-install-peers

**Review by scissorsneedfoodtoo — APPROVED** [review]
Tested locally with pnpm v8 and everything LGTM.

**Comment by Sboonny** [comment]
gitpod version is still 7.22.0

**Comment by raisedadead** [comment]
> gitpod's pnpm version is still 7.22.0, is the plan to update it in different PR?

We are not pinning versions in GitPod. For a consistent and predictable environment, we should do that for all our tooling MongoDB, Node.js, etc.

I am not using it actively, so @ShaunSHamilton is better suited here.

But IMHO, we should build and publish a docker image (not a Dockerfile) with the tooling frozen that the instances are spun up from.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**